src/components/environment-banner/environment-banner.jsx

```diff
@@ -40,11 +41,11 @@ export default class EnvironmentBanner extends React.Component {
       && !this.state.dismissed;
 
     return showBanner ? (
-      <section className='environment-banner bg-warning'>
+      <section styleName='environment-banner' className='bg-warning'>
         <div className='container-fluid'>
           <p>Environment: {this.state.apiEnvironment}</p>
           {/* Uncomment to make dismissible
```

Comment:
@Mr0grog do we actually still want that code that is commented out or should that be culled?

Reply:
Yeah, this is a good opportunity to cut that out. I actually didn’t realize it was there and not sure why I merged it with that in the first place. This banner should probably never be dismiss-able.

Reply:
@g-merrill do you think you can remove the commented out code and also the `environment-banner-close-x` styles then? Thanks!